convert/jenkinsjson/json/readMavenPom.go

```diff
@@ -0,0 +1,23 @@
+package json
+
+import (
+	harness "github.com/drone/spec/dist/go"
+)
+
+func ConvertReadMavenPom(node Node) *harness.Step {
+
+	settings := map[string]interface{}{
+		"pom_path": node.ParameterMap["file"],
```

Comment:
Is this always a single entry and not a list of files?

Reply:
https://www.jenkins.io/doc/pipeline/steps/pipeline-utility-steps/#readmavenpom-read-a-maven-project-file Single entry, exact file